server/src/controllers/cars/addCar.ts

```diff
@@ -0,0 +1,14 @@
+import { Request } from 'express';
+import { addCarService } from '../../services';
+import { updateCarSchema } from '../../validation';
+
+const addCar = async (req: Request) => {
+  const customerId = req.cookies.id;
+  const { body } = req;
+  await updateCarSchema.validate(body, customerId);
```

Comment:
```suggestion
  await updateCarSchema.validate(body, customerId);
```
Does validate accept two params?